### ISSUE TITLE
Upgrade Problem Wordpress 4.7

### ISSUE THREAD

**Issue by cemmildon** [ProjectNami/projectnami#156]
I upgraded new version of Wordpress in Azure. New themes came but didn't upgrade to 4.7. And now is slow.

**Comment by patrickebates** [comment]
So how exactly did you upgrade?  We haven't completed our merge with 4.7 yet, so we haven't published any updates.

**Comment by cemmildon** [comment]
Wordpress opened new notification to me. And I did update.

**Comment by patrickebates** [comment]
Did you download the files yourself and apply them to the site?  I ask because we have the built-in updater disabled to prevent this from happening.

If you downloaded the files manually, you should download our ver 1.4.3 and apply it over your install.  We are working on merging the 4.7 updates into what will become ver 1.5.0

**Comment by cemmildon** [comment]
No I watched your video. And I did your sentences in my Azure. Then new themes came to Dashboard but didn't update to new version of Wordpress. How Can I fix them ?

**Comment by cemmildon** [comment]
Can you hear me?

**Comment by patrickebates** [comment]
Yes, but not sure that I follow.  Sounds like you were able to upgrade the themes, and you're waiting on us to update PN with 4.7, which is in progress.

Are you indicating that updating the themes caused them to stop working?

**Comment by cemmildon** [comment]
No. My theme is same. Wordpress'es new themes came to me. And I removed. So WP didn't update clearly. Just new themes came. Everything is as same as 4.6.

**Comment by talisman3** [comment]
Hi. I am an Azure using Project Nami application deployment of WordPress on MSSQL. What are the authorization details to be used from Azure to retrieve the latest Project Nami updates using the "Continuos Delivery (Preview)" option. When requested to configure the source I have "Source Code Repository", "Build - configure continuous delivery". 

For Deployment Source I buggered up (I think) by using my github credentials. 

Basically all I need is a how-to guide on setting up the project nami source as my repository for keeping up to date with updates.

Thanks

**Comment by patrickebates** [comment]
http://projectnami.org/video-updating-project-nami/

See the notes at the bottom of that post.

The secret sauce is selecting External Repository rather than GitHub, and setting branch Latest.
